### ISSUE TITLE
CLI Install issues in the Linux Subsystem for Windows

### ISSUE THREAD

**Issue by swgriffith** [Azure/azure-xplat-cli#2805]
CLI Version: Installing Latest
OS Type: Win - Linux Subsystem for Windows
Installation via:  npm 

Mode: N/A - Install issue

Description:
Trying to install the CLI under the Linux Subsystem for Windows.  Ran through the install steps, with mixed success.  Seems to have installed node, but when I run 'node -v' I get no version back.  Running 'npm install azure-cli -g' runs for a while and seems like its going to succeed, but then ultimately does not install successfully. I suspect the root cause may be in the node installation.

**Comment by eduardkoller** [comment]
@swgriffith - Looks like the node and/or npm install may be the culprit.
Are you able to successfully run/install other packages?

**Comment by swgriffith** [comment]
When I initially set up LSW I did install GCC and compiled a basic hello world app. That worked fine. I have also tried to install docker, which did not succeed. Node and npm seem to have installed, but not completely, as some commands work and others don’t. This is almost certainly an issue with the node and npm installation rather than anything wrong with the cli install. Just thought I'd register it in case anyone has tried and run into the same issues and worked through it.

**Comment by eduardkoller** [comment]
@russalex - per our email discussion, this seems to be a more general issue related to node on bash on windows. Can you please update this issue as soon as it gets fixed?

**Comment by amarzavery** [comment]
@swgriffith - I tried this right now and the installation process works seamlessly. 

Install node.js on win10 bash using the instructions specified for ubuntu from this [doc](https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions).

Once you have installed node.js then `sudo npm install -g azure-cli`. As of today it should install the latest version `0.10.4` version of the CLI and it should work fine.

I tried it on my box and it works well.
```bash
admin@MyMachine:/mnt/d/sdk$ azure -v
0.10.4
admin@MyMachine:/mnt/d/sdk$
```
